package.json

```diff
@@ -144,8 +146,10 @@
     "eslint-plugin-react": "^7.21.5",
     "framer-motion": "^2.9.4",
     "husky": "^4.3.0",
+    "identity-obj-proxy": "^3.0.0",
     "jest": "^26.6.0",
     "jest-junit": "^12.0.0",
+    "jest-watch-typeahead": "^0.6.1",
```

Comment:
interesting...

Reply:
really handy tool when you want to execute only subset of the tests. Have found this lib in `create-react-app`